vertx-web/src/main/java/io/vertx/ext/web/handler/impl/CorsHandlerImpl.java

```diff
@@ -28,6 +28,7 @@
 import java.util.Set;
 import java.util.regex.Pattern;
 
+import static io.netty.handler.codec.http.HttpHeaderNames.VARY;
```

Comment:
Side-question: Would it make sense to add the `VARY` constant to `io.vertx.core.http.HttpHeaders` for consistency?

Reply:
This is a fix needed on core, see: https://github.com/eclipse-vertx/vert.x/pull/3489
only after it is merged I can use the "vertx" headers

Reply:
Ah, I see - I failed to notice that. 👍